src/tss2-esys/esys_crypto_ossl.c

```diff
@@ -660,6 +661,8 @@ iesys_cryptossl_pk_encrypt(TPM2B_PUBLIC * pub_tpm_key,
                    "Could not set RSA passing.", cleanup);
     }
 
+/* LibreSSL does not provide these OAEP functions */
+#ifdef EVP_PKEY_CTX_set0_rsa_oaep_label
```

Comment:
Should we check if both EVP_PKEY_CTX_set_rsa_oaep_md and EVP_PKEY_CTX_set0_rsa_oaep_label are defined instead of wrapping the two calls in one check? Something like:
#if defined(EVP_PKEY_CTX_set_rsa_oaep_md) && defined(EVP_PKEY_CTX_set0_rsa_oaep_label)

Reply:
Also together with (void)label;
 goto_error(r, TSS2_ESYS_RC_NOT_IMPLEMENTED,
                   "RSA OAEP not implemented.", cleanup);
 should be added, because the secret encryption in StartAuthSession will not be possible. The integration tests with secret encryption will fail. But instead of "0x2c4 TPM_RC_VALUE" the message "RSA OAEP not implemented" would be  displayed in the log of the test.

Reply:
OK, I will update the change to reflect these comments. Thanks.